src/Workspaces/Core/Portable/Options/IDocumentOptions.cs

```diff
@@ -1,12 +1,14 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
+
 namespace Microsoft.CodeAnalysis.Options
 {
     /// <summary>
     /// Returned from a <see cref="IDocumentOptionsProvider"/>
     /// </summary>
     interface IDocumentOptions
     {
-        bool TryGetDocumentOption(OptionKey option, out object value);
+        bool TryGetDocumentOption(OptionKey option, out object? value);
```

Comment:
Good question, but it's actually not correct! null could be a valid value of a document option, so it can be null even if it does return true. I should add a comment.